theme/imageuploadicon.css

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (c) 2003-2018, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+:root {
+	--ck-image-upload-icon-size: 1.5em;
+	--ck-image-upload-icon-width: 2px;
+
+	--ck-color-image-upload-icon: hsl(0, 0%, 100%);
+	--ck-color-image-upload-icon-background: hsl(120, 100%, 27%);
+}
+
+.ck-image-upload-complete-icon {
```

Comment:
I see the wireframe styles (display, position) mixed up with pure visual styles (background, animation) here. The later should be moved to Lark.